src/main/configurations/morcoreMessageProcessor/Configuration_FromMorCore.xml

```diff
@@ -50,7 +50,7 @@
         <Receiver name="Frank_GetTaakType">
             <ApiListener
                 name="Frank_GetTaakType"
-                uriPattern="/v1/taaktypes/{uuid}"
+                uriPattern="/v1/taaktype/{uuid}"
```

Comment:
Needs to be /api/v1/taaktype/{uuid}. The "/api" section was added in a recent update. Make sure your branch is up-to-date.